docker-compose.yaml

```
@@ -0,0 +1,29 @@
+version: "3"
+
+services:
+   mysql:
+      image: mysql:latest
+      container_name: mysql-db
+      restart: always
+      environment:
+         MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-4DM1N@_2O2A}
+         MYSQL_DATABASE: ${MYSQL_DATABASE:-ROOT}
+         MYSQL_USER: ${MYSQL_USER:-ADMIN}
+         MYSQL_PASSWORD: ${MYSQL_PASSWORD:-4DM1N@_2O2A}
```

já que esse valores serão os default e ficaram _hard coded_, talvez fique mais práticos valores simples
```suggestion
         MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-root}
         MYSQL_DATABASE: ${MYSQL_DATABASE:-db}
         MYSQL_USER: ${MYSQL_USER:-admin}
         MYSQL_PASSWORD: ${MYSQL_PASSWORD:-admin}
```